analysis.ml

```diff
@@ -1,34 +1,44 @@
 open Instr
 
-let successors (instrs : instruction_stream) pc =
+let pcs (instrs : instruction_stream) : pc array =
+  Array.mapi (fun pc _ -> pc) instrs
```

Comment:
Btw. you probably mean in the pretty printer / scope inference. There you talk about line numbers but they are 0-based. In some cases you pretty-print the code with the pc, so its not that bad. But in other cases (eg. when the annotation is too small) you just say: "n : Error: Variable x is not declared." and n is a pc, but not a line number.